naplib/naplab/process_ieeg.py

```diff
@@ -122,34 +123,29 @@ def process_ieeg(
     store_all_wav : bool, default=False
         If True, store all recorded wav channels that were stored by the neural recording hardware. This may include
         any other signals that were hooked up at the same time, such as EKG, triggers, etc.
-    aud_fn : Optional[Union[str, Callable]], default='default'
-        Function for computing spectrogram from trial stimulus sounds. If None or 'none', no spectrograms will be
-        computed. If 'default', ``naplib.features.auditory_spectrogram`` will be used. If a callable ``f``, the
-        function will be applied to each stimulus audio and should have signature
-        ``(x: NDArray, sr: float, **aud_kwargs) -> NDArray``, where ``x`` is 1-D audio signal with shape
-        (in_samples,) and ``sr`` is the sampling rate of the audio. The returned array (spectrogram) should
-        have shape (n_samples, freq_bins).
-    aud_kwargs : dict, default={}
-        Keyword arguments to pass to ``aud_fn``. In case of the default function ``naplib.features.auditory_spectrogram``,
-        it can include overrides for frame_len, tc, and factor.
+    aud_fn : optional callable or dict, default=naplib.features.auditory_spectrogram
+        Function for computing spectrogram from trial stimulus sounds. If None, no spectrograms will be computed.
+        By default, `naplib.features.auditory_spectrogram` will be used. If a callable `f`, the function `f` will be
+        applied to each stimulus audio and should have signature `(x: NDArray, sr: float, **kwargs) -> NDArray`,
+        where `x` is 1-D audio signal with shape (in_samples,) and `sr` is the sampling rate of the audio. The returned
+        array (spectrogram) should have shape (n_samples, freq_bins). If a dictionary, the keys will be used in field
```

Comment:
maybe just change to "with shape (n_samples, m_features)" since it doesn't have to be a spectrogram

Reply:
I modified the docstrings a bit more than that to avoid implying that it has to be a spectrogram.